Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:buster-slim
+FROM docker.io/debian:buster-slim
```

Comment:
Please don't specify the registry.

Reply:
Why not? I think it's best to be explicit about where we're pulling from so we don't end up with weird edgecases if someone is using a different set of registries (isn't that the point of OCI, after all?)

Reply:
Because this gets deployed to an environment where it might pull from an internal registry. I'm not too worried about `debian:buster-slim` in common registries that might be pulled from, but if it does become an issue, the better procedure would be to specify a digest. We can still get weird edge cases with mutable tags and pull caches, especially for clients like docker.

Reply:
https://www.redhat.com/en/blog/be-careful-when-pulling-images-short-name

Reply:
Why prefer `registry/repo:tag` over `repo:tag@digest`?

Reply:
If we specify the digest, we're not going to get updated images.... `buster-slim` is updated periodically. Also, it will mean hardcoding the digest for _one particular architecture_ which is... not ideal.

Reply:
It doesn't lock to one architecture. Image indexes have digests too.

I suppose docker.io is alright, but I may dig into OKD's caching later.